src/includes/performance/distributed-tracing/dotnet.mdx

```diff
@@ -0,0 +1,49 @@
+To obtain a trace header from the span, use `ISpan.GetTraceHeader()` method, then pass it to the downstream service. If the communication happens over HTTP, it is recommended that you set the value to the `sentry-trace` HTTP header.
+
+```csharp
+using System;
+using System.Net.Http;
+using Sentry;
+
+var transaction = SentrySdk.StartTransaction("name", "op");
+var sentryTraceHeader = transaction.GetTraceHeader();
+
+using var request = new HttpRequestMessage(HttpMethod.Get, "https://example.com");
+request.Headers.Add("sentry-trace", sentryTraceHeader.ToString());
+```
+
+<Note>
+
+Sentry also provides `SentryHttpMessageHandler`, which can be used to [automatically attach trace headers to outgoing HTTP requests](/platforms/dotnet/performance/included-instrumentation/).
+
+</Note>
```

Comment:
What if we point out it's out of the box before we show the snippet?

We don't just recommend doing this for HTTP, we recommend for any system communication which the user would like visibility end-to-end. Specifically for HTTP we provide `SentryHttpMessageHandler` so lets point them to it, and after that demonstrate how to programatically do it:


````suggestion
To obtain a trace header from the span, use `ISpan.GetTraceHeader()` method, then pass it to the downstream service.

<Note>

For HTTP communications, Sentry's SDK providers `SentryHttpMessageHandler`, which can be used to [automatically attach trace headers to outgoing HTTP requests](/platforms/dotnet/performance/included-instrumentation/).

</Note>

The code snippet below demonstrates propagating the trace programatically. Even though `SentryHttpMessageHandler` does that automatically for your, this illustrates the mechanism so it can be replicated in other protocols and technologies such a messaging systems.

```csharp
using System;
using System.Net.Http;
using Sentry;

var transaction = SentrySdk.StartTransaction("name", "op");
var sentryTraceHeader = transaction.GetTraceHeader();

using var request = new HttpRequestMessage(HttpMethod.Get, "https://example.com");
request.Headers.Add("sentry-trace", sentryTraceHeader.ToString());
```

````